backend/server.js

```diff
@@ -1,7 +1,12 @@
 const express = require('express'); 
 const app = express();
+const cors = require('cors');
 const port = 8080;
 app.use(express.json()); 
+app.use(cors());
+
+const usernameRoute = require('./routes/usernameRoute');
```

Comment:
keep your imports/requires towards the top with the other imports

Reply:
done